src/forms/form-group.scss

```diff
@@ -14,6 +14,13 @@
   // stylelint-disable-next-line primer/spacing
   margin: 15px 0;
 
+  // Autocomplete with embedded icon
+  .form-control.autocomplete-embedded-icon-wrap {
+    &:focus-within {
+      background-color: var(--color-canvas-default);
+    }
+  }
+
```

Comment:
When an input in a `.form-group` is focused, it changes colors. We need to make sure that color of the parent embedded icon container also changes.

This fixes how the color is not uniform in this screenshot : https://github.com/github/github/pull/214426#discussion_r835613107